classes/tests/test_world.py

```diff
@@ -0,0 +1,24 @@
+import unittest
+
+from classes import World
+
+
+class EventMock:
+    def __init__(self, time):
+        self.time = time
+
+    def perform(self, world):
+        world.time = self.time
+
+
+class WorldTestCase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.world = World(40)
+
+    def test_run(self):
+        self.world.stack = [EventMock(time) for time in range(10, 41, 10)]
+        self.world.run()
```

Comment:
This can be changed if the stack is initialized in the __init__ method

Reply:
yes, but I think it is nice to have a test that does only test the